src/rctrando_init.ts

```diff
@@ -22,7 +29,7 @@ let initedMenuItems:boolean = false;
 let subscriptions = []
 
 const minApiVersion = 52;// or 60?
-const targetApiVersion = 77;// v0.4.5
+const targetApiVersion = 84;// v0.4.11
```

Comment:
v0.4.19.1 is 104, so whenever you feel like updating OpenRCT2 this could be updated

```suggestion
const targetApiVersion = 104;// v0.4.19.1
```